app/pages/_document.tsx

```diff
@@ -14,6 +18,7 @@ class MyDocument extends Document {
         <body>
           <Main />
           <NextScript />
+          {CONFIG.ENABLE_ANALYTICS && <Analytics />}
```

Comment:
The environment variable is going to be a string (see below)
```suggestion
          {CONFIG.ENABLE_ANALYTICS === 'true' && <Analytics />}
```